.github/workflows/test.yml

```diff
@@ -139,6 +139,8 @@ jobs:
       env:
         CIBW_PLATFORM: pyodide
 
-    - name: Run the emulation tests
+    - name: Run tests with 'CIBW_PLATFORM' set to 'pyodide'
       run: |
-        pytest test/test_emscripten.py
+        python ./bin/run_tests.py
+      env:
+        CIBW_PLATFORM: pyodide
```

Comment:
Strange... On ubuntu 20.04 the slowest two tests are:
```
703.00s call     test/test_before_test.py::test
626.81s call     test/test_testing.py::test_extras_require
```
We skip `test_before_test` so that saves 11 minutes by itself. But `test_extras_require` only takes 45 seconds! For comparison on osx `test_extras_require` takes 210 seconds. I wonder how the time breaks down between building and testing. Probably mostly building?

Reply:
I just did a little experiment locally. Most of our tests don't involve testing, but for the ones that do, the time is spent roughly 70% in the testing phase. I'd _love_ our test suite to be faster, but it's a lot of shelling out to external tools, which in turn do lots of I/O, so gains are hard to come-by.